### ISSUE TITLE
Lens no longer connects to cluster properly after upgrade

### ISSUE THREAD

**Issue by blairham** [lensapp/lens#897]
**Describe the bug**
None of my clusters connect and I can not see anything

**To Reproduce**
1. Start lens, view nothing

**Expected behavior**
When starting I would expect to see pods and namespaces and pretty much any resource. I did not make any changes to my kubecontext before or after the forced upgrade

**Screenshots**
![image](https://user-images.githubusercontent.com/5043780/93365695-1c8b0d80-f818-11ea-9b0a-809dfa199508.png)

**Environment (please complete the following information):**
- Lens Version: 3.6.0
- OS: [e.g. OSX] 10.15.6
- Installation method (e.g. snap or AppImage in Linux): homebrew on macOS
 
**Logs:**
When you run the application executable from the command line you will see some logging output. Please paste them here:
```
info: [IPC]: setup channel "cluster:activate"
info: [IPC]: setup channel "cluster:disconnect"
info: [IPC]: setup channel "cluster:install-feature"
info: [IPC]: setup channel "cluster:uninstall-feature"
info: [IPC]: setup channel "cluster:upgrade-feature"
info: 🚀 Starting Lens from "/Users/blair/Library/Application Support/Lens"
info: Checking for update
info: [STORE]: LOADED from /Users/blair/Library/Application Support/Lens/config.json
info: [STORE]: LOADED from /Users/blair/Library/Application Support/Lens/lens-cluster-store.json
info: [STORE]: LOADED from /Users/blair/Library/Application Support/Lens/lens-workspace-store.json
info: [CLUSTER-MANAGER]: init cluster {"id":"040a9a22-5642-47d2-9688-10b408275465","name":"development","initialized":false}
info: [CLUSTER]: "development" init success {"id":"040a9a22-5642-47d2-9688-10b408275465","context":"development","apiUrl":"https://loft.hqo.dev/kubernetes/cluster/development"}
info: LensProxy server has started at http://localhost:53467
info: Update for version 3.6.0 is not available (latest version: 3.6.0, downgrade is disallowed).
info: [KUBE-AUTH]: out-channel "kube-auth:040a9a22-5642-47d2-9688-10b408275465" {"data":"Authentication proxy started\n","meta":{"id":"040a9a22-5642-47d2-9688-10b408275465","name":"development","initialized":true}}
info: [CLUSTER]: activate {"id":"040a9a22-5642-47d2-9688-10b408275465","name":"development","initialized":true}
info: [CLUSTER]: bind events {"id":"040a9a22-5642-47d2-9688-10b408275465","name":"development","initialized":true}
info: [CLUSTER]: refresh {"id":"040a9a22-5642-47d2-9688-10b408275465","name":"development","initialized":true}
error: failed to request selfSubjectAccessReview: HttpError: HTTP request failed
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
 apiVersion: v1
clusters:
- cluster:
    server: https://loft.hqo.dev/kubernetes/cluster/development
  name: loft-development
contexts:
- context:
    cluster: loft-development
    user: loft-development
  name: development
current-context: development
kind: Config
preferences: {}
users:
- name: loft-development
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - token
      - --silent
      - --config
      - /Users/blair/.loft/config.json
      command: /Users/blair/.devspace/plugins/NB2HI4DTHIXS6Z3JORUHKYROMNXW2L3MN5THILLTNAXWY33GOQWWIZLWONYGCY3FFVYGY5LHNFXA0000/binary
      env: null
```

**Additional context**

**Comment by crackoffnet** [comment]
Same for me

**Comment by crackoffnet** [comment]
Problem solved for me with upgrading to v3.6.2

**Comment by jakolehm** [comment]
Should be fixed in v3.6.2.
